virtual-setup/zoom-procedures.md

```diff
@@ -1 +1,197 @@
+You can toggle mute on and off using the `Mute` (or `Unmute` if you are muted) icon on the left side of the controls at the bottom of the Zoom window.
+
+<img src = screenshots/zoom-participant-mute.png>
+
+Where possible, please utilize headphones throughout training to minimize audio feedback.
+
+### Video
+
+If you are experiencing issues with the call quality, turning off your video may help.
+You can toggle video on and off using the Stop Video icon (or Start Video if video is off) on the left side of the controls at the bottom of the Zoom window.
+
+<img src = screenshots/zoom-participant-video.png>
+
+### Interacting with instructors and providing feedback
+
+#### Participant pane
+
+The participant pane will both allow you to identify the host and co-hosts of the meeting (CCDL staff) and to provide feedback during instruction. 
+
+To bring up the participant pane, click the participants icon in the middle of the controls at the bottom of the Zoom window.
+
+<img src = screenshots/zoom-participant-participant-icon.png>
+
+The participant pane will appear on the side of the main Zoom window and look something like this:
+
+<img src = screenshots/zoom-participant-panel.png height = 500>
+
+There are feedback controls on the bottom of the participant pane that will allow you to answer yes or no questions, give feedback on the speed of instruction, and under `more` you can indicate if you need a break.
+
+<img src = screenshots/zoom-participant-feedback.png width = 400>
+
+#### Chat
+
+To bring up the chat window to ask questions during instruction, click the chat icon in the middle of the controls at the bottom of the Zoom window.
+
+<img src = screenshots/zoom-participant-chat.png>
+
+The chat pane will appear below the participant pane. 
+
+<img src = screenshots/zoom-chat-pane.png height = 500>
+
+**You should direct your questions to the host of the meeting** by selecting their name from the dropdown menu of the Chat window.
+
+<img src = screenshots/zoom-chat-dropdown.png width = 300> 
+
+Alternatively, you can hover over the Host's name in the participant list and click `More` and select `Chat` from the dropdown menu.
+
+<img src = screenshots/zoom-participant-hover-more.png width = 400>
+
+<img src = screenshots/zoom-participant-more-chat.png width = 300>
+
+## Joining a Zoom call
+
+CCDL staff will provide you with a link to join a Zoom call for each day during training.
+_Please do not share this link._
+
+On the first day of instruction, we will ask you to introduce yourself on the Zoom call so please be ready with computer audio (more on that below).
+
+### To join a call
+
+Follow the Zoom link you've received.
+It should open in your browser.
+If you have Zoom installed on your computer, you will be prompted to start the call from the Zoom application and can click the `Open link` or `OK` button. (_Note: these screen shots were taken on a Mac using Firefox, your interface may look slightly different._)
```

Comment:
We should probably grab a Chrome snapshot, as that is what most will be using, and it looks very different.